lib/entities/entity_factory.cpp

```diff
@@ -56,7 +53,7 @@ int EntityFactory::createWeapon(double xPos, double yPos) const{
     comps->push_back(std::make_unique<PhysicsComponent>(50, 0, 0, 0, true, Kinematic::IS_NOT_KINEMATIC, true, true));
     comps->push_back(std::make_unique<TextureComponent>(std::move(r)));
     comps->push_back(std::make_unique<PickupComponent>());
-    //comps->push_back(std::make_unique<WeaponComponent>(1, 1, 10, 5));
+    // comps->push_back(std::make_unique<WeaponComponent>(1, 1, 10, 5));
```

Comment:
?

Reply:
Not intentional, Luuk quoted: "Een fout van mij". This will be reverted in the next commit.